website_sale_delivery_date/readme/CONTEXT.md

```
@@ -0,0 +1,47 @@
+## Example Use Cases
+
+### Example 1: Delay Type = Hours
+
+**Configuration:**
+- Available delivery days: Monday to Friday
+- Delivery window: 11:00 to 16:00
+- Minimum delivery delay: 11 hours
+- Earliest delivery time of day: 11:00
+
+**Results:**
+- An order placed Monday at 09:00 can be delivered earliest Tuesday at 11:00
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)